src/Forms/GridField/GridFieldFilterHeader.php

```diff
@@ -330,19 +350,6 @@ public function getSearchFieldSchema(GridField $gridField)
         return json_encode($schema);
     }
 
-    private function getTitle(GridField $gridField, object $inst): string
-    {
-        if ($gridField->Title) {
-            return $gridField->Title;
-        }
-
-        if (ClassInfo::hasMethod($inst, 'i18n_plural_name')) {
-            return $inst->i18n_plural_name();
-        }
-
-        return ClassInfo::shortName($inst);
-    }
```

Comment:
Moved this method to conform with the [class member ordering](https://docs.silverstripe.org/en/5/contributing/coding_conventions/php_coding_conventions/#class-member-ordering) standard